src/source-control/GitSourceControl.ts

```diff
@@ -198,6 +198,33 @@ export class GitSourceControl implements SourceControl {
     this.repo = ourRepository;
   }
 
+  private getLLBFromConfig(): BranchName[] {
+    const configFileContents = fs
+      .readFileSync(`${this.repoPath}/sttack.config.json`)
+      .toString();
+    const { longLivedBranches } = JSON.parse(configFileContents);
+    return longLivedBranches;
+  }
+
+  async getMergeCommitByCommitHash(commit: Commit): Promise<CommitHash[]> {
+    const repo = await nodegit.Repository.open(this.repoPath);
+    const longlivedbranches: BranchName[] = this.getLLBFromConfig(); //TODO: Get long lived branches from sttack.config.json
```

Comment:
```suggestion
    const longlivedbranches: BranchName[] = this.getLongLivedBranchesFromConfig();
```

Probably need to run Prettier on this